OpenEdXMobile/src/main/java/org/edx/mobile/module/analytics/Analytics.java

```diff
@@ -603,6 +604,23 @@ void trackOpenInBrowserBannerEvent(@NonNull String eventName, @NonNull String bi
      */
     void trackProfileScreenEvent(@NonNull String eventName, @NonNull String biValue);
 
+    /**
+     * Track event triggered click on Video download Quality Modal
+     *
+     * @param eventName Name of the event
+     * @param biValue   BiValue of the event
+     * @param courseId  course id of the course currently viewing
+     */
+    void trackVideoDownloadQualityClicked(String eventName, String biValue, String courseId);
```

Comment:
Can we use `@Nullable` annotation with `courseId`?

Reply:
`courseId ` remove as per new decision.